source/query/atlas-search.txt

```diff
@@ -12,11 +12,26 @@ Run Atlas Search Queries
    :depth: 1
    :class: singlecol
 
-You can run Atlas Search :pipeline:`$search` and :pipeline:`$searchMeta`
-queries from |compass| if you :ref:`created the Atlas Search index
-<compass-create-search-index>` for data on the Atlas cluster. You can
-also run :pipeline:`$vectorSearch` queries from |compass| if you have an
-existing Atlas Vector search index on your data in Atlas. 
+Atlas Search queries are run as an :ref:`aggregation pipeline stage 
+<compass-agg-builder>`. If you have an existing Atlas Search Index for data on 
+your Atlas cluster, you can run :pipeline:`$search` and :pipeline:`$searchMeta` 
+queries. If you have an  existing Atlas Vector search index on your data in 
+Atlas, you can also run :pipeline:`$vectorSearch` queries. 
+
+Before You Begin 
+----------------
+
+Create an :ref:`Atlas Search index <compass-create-search-index>` or an 
+:ref:`Atlas Vector Search index <compass-create-vector-search>`.
+
+.. note:: 
+   
+   Atlas Search index management in the |compass-short| UI is available only 
+   for: 
+   
+   - Atlas local deployments.
+   - M10+ clusters.
```

Comment:
This restriction actually doesn't exist anymore -- you can manage search indexes on free/shared tier clusters in Compass. Would you mind removing this restriction as a drive-by fix in this PR?

Reply:
Can do!

Reply:
You won't be able to manage Atlas Search indexes in Data Explorer